secator/query/_base.py

```
@@ -21,6 +21,18 @@ def __init__(self, workspace_id: str, config: Optional[dict] = None, context: Op
 		self.config = config or {}
 		self.context = context or {}
 
+	def list_workspaces(self) -> List[Dict[str, Any]]:
+		"""List all workspaces accessible via this backend."""
+		return []
+
+	def get_workspace(self, workspace_id: str) -> Optional[Dict[str, Any]]:
+		"""Get info for a specific workspace."""
+		return None
+
+	def list_runners(self, workspace_id: str = None, runner_type: str = None) -> List[Dict[str, Any]]:
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)